codecov/__init__.py

```diff
@@ -925,13 +925,11 @@ def main(*argv, **kwargs):
                 True,
                 dont_search_here,
             ):
-                cmd = (
-                    sanitize_arg("", codecov.gcov_exec or "")
-                    + " -pb "
-                    + sanitize_arg("", codecov.gcov_args or "")
-                    + " "
-                    + path
-                )
+                cmd = sanitize_arg("", codecov.gcov_exec or "").split(" ")
+                cmd.append("-pb")
+                if codecov.gcov_args:
+                    cmd.append(sanitize_arg("", codecov.gcov_args or ""))
+                cmd.append(path)
```

Comment:
```suggestion 
                cmd = [
                    *sanitize_arg("", codecov.gcov_exec or "").split(" "),
                    "-pb",
                    sanitize_arg("", codecov.gcov_args or ""),
                    path,
                  ]
```

Reply:
That is how I initially had it, but my CI threw some errors with "" entries if gcov_args not supplied. It was trying to find "" as an input file.

Also I split gcov_exec due to "llvm-cov-6.0 gcov" possibly being passed in.